docs/UserGuide.md

```diff
@@ -143,6 +144,7 @@ Format: `editp INDEX (must be a positive integer) [n/NAME] [p/PHONE] [e/EMAIL] [
     - have each domain label consist of alphanumeric characters, separated only by hyphens, if any.
 * The length of the `ADDRESS` must be between 6 and 70 characters
 * The length of the `REMARK` must be less than or equal to 70 characters
+* The length of a tag must be less than 70 characters.
```

Comment:
Maybe should write less than or equals to. I remember there being an issue with unclear specifications for this

Reply:
Will do!